modules/customize-certificates-api-add-named.adoc

```diff
@@ -18,11 +18,24 @@ client's URL.
 reach the API server.
 * The certificate must have the `subjectAltName` extension for the URL.
 
+
 [WARNING]
 ====
 Do not provide a named certificate for the internal load balancer (host
 name `api-int.<cluster_name>.<base_domain>`). Doing so will leave your
 cluster in a degraded state.
+
+If a certificate chain is used, all certificates must be manually concatenated into a single named certificate file.
+
+Ex: (cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem) 
```

Comment:
Can you move this to be with the above sentence instead on its own line. And word it something like: 

```
For example, you can do this using a command like the following: `cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem`.
```